pkg/patch/sidecar.go

```diff
@@ -42,6 +50,28 @@ const sidecarContainerTemplate = `
   }
 }
 `
+const xrayDaemonContainerTemplate = `
+{
+  "name": "xray-daemon",
+  "image": "amazon/aws-xray-daemon",
+  "securityContext": {
+    "runAsUser": 1337
+  },
+  "ports": [
+    {
+      "containerPort": 2000,
+      "name": "xray",
+      "protocol": "UDP"
+    }
+  ],
+  "resources": {
+    "requests": {
+      "cpu": "32",
```

Comment:
```suggestion
      "cpu": "{{ .CpuRequests }}",
```

Reply:
If it is CpuRequests then I think it will be in addition to what Envoy needs (i.e. double). Those numbers were given by x-ray team but not sure if there is any data behind them. I will find out, for now I will set it to "cpu=32m", "memory=256mi".